libsolidity/interface/AssemblyStack.h

```diff
@@ -63,6 +63,9 @@ class AssemblyStack
 	/// Run the assembly step (should only be called after parseAndAnalyze).
 	eth::LinkerObject assemble(Machine _machine);
 
+	/// Run the assembly step (should only be called after parseAndAnalyze) and print source.
+	std::string assemblySource(Machine _machine);
```

Comment:
Basically there came up the need to return an assembly source as opposed to bytecode. Perhaps we could introduce `AssemblyMachineObject`, which includes the assembly source and the what `LinkerObject` contains (all the fields are optional).